next/src/field/schema.ts

```diff
@@ -19,6 +20,30 @@ function getJsonType(schema: NonBooleanJsfSchema): string {
   return 'text'
 }
 
+/**
+ * Get the input type for a field
+ * @param schema - The non boolean schema of the field
+ * @returns The input type for the field, based schema type. Default to 'text'
+ */
+function getInputType(schema: NonBooleanJsfSchema): string {
+  const presentation = schema['x-jsf-presentation']
+  if (presentation?.inputType) {
+    return presentation.inputType
+  }
+
+  const schemaType = schema.type
+
+  if (schemaType === 'string') {
+    return 'text'
+  }
+
+  if (schemaType === 'number') {
+    return 'number'
+  }
+
+  return 'text'
```

Comment:
I'm not sure if not having `x-jsf-presentation` is a common scenario but I feel this is still not 100% safe. I can think of 2 cases:
- when `schemaType` is `boolean`. In this case it should map to checkbox, no?. As it stands it will default to text.
- there might be cases where type is `[string, null]`. Do you want to cover this case and return `text` when it happens?

Reply:
all good points @joaoalmeida-remote 👍

Reply:
@joaoalmeida-remote feel free to re-review :)